cfgov/v1/views.py

```diff
@@ -147,7 +147,13 @@ def login_with_lockout(request, template_name='wagtailadmin/login.html'):
     """
     Displays the login form and handles the login action.
     """
-    redirect_to = request.GET.get(REDIRECT_FIELD_NAME, '')
+    redirect_to = request.POST.get(REDIRECT_FIELD_NAME,
```

Comment:
reverted my original change here, realized i introduced a bug where it would lose the redirect url during the Post request